ansible/webscale.yml

```diff
@@ -7,4 +7,4 @@
       include_tasks: tasks/validation.yml
   roles:
     - operators
-  
+    - kafka-logging-setup
```

Comment:
Let's add a tag here so we can selectively skip if needed. In cases where we only want to run some comonponents of the webfuse architecture. Let's do that going forward for all roles.